config/initializers/okcomputer.rb

```diff
@@ -80,6 +80,26 @@ def check
                                 OkComputer::SizeThresholdCheck.new(queue, 20) { Resque::Failure.count(queue) }
 end
 
+# check for the right number of workers
+class WorkerCountCheck < OkComputer::Check
+  def check
+    if Resque.workers.first.to_s.include?('stage')
```

Comment:
Could we add the expected number of workers to settings.yml?

Reply:
https://github.com/sul-dlss/shared_configs/pull/1084 and https://github.com/sul-dlss/shared_configs/pull/1085 to add a worker count to settings.